examples/evm-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -15,16 +15,16 @@ if (!privateKey) {
   throw new Error("Missing environment variable: PRIVATE_KEY");
 }
 
-const SEPOLIA_CHAIN_ID = 11155111;
-const AMOY_CHAIN_ID = 84532;
+const SOURCE_CHAIN_ID = 11155111;
+const DESTINATION_CHAIN_ID = 84532;
```

Comment:
The id explicitly corresponds to the name of the chain. As a person looking at the example, the numbers would lead to the question of what they mean and where you can look at them. So mentioning the chains is important, at least this way:
``` ts
const SOURCE_CHAIN_ID = 11155111; // Sepolia
const DESTINATION_CHAIN_ID = 84532; // Amoy
```